### DISCUSSION TITLE
Data loading error

### DISCUSSION THREAD

**Discussion by NoelLeber** [visualize-admin/visualization-tool#766] [General]
While creating a New Cube everything went well. I could publish it without any problem. But when i open the Cube in Visualize there is a Error Message saying: 
<img width="682" alt="Unbenannt" src="https://user-images.githubusercontent.com/113584933/194510206-fd73c3f2-51be-43ad-9c97-ad05a4c74785.PNG">

I don't quit understand what this means, also I couldn't find an Error in the Cube Designer.

**Comment by bprusinowski** [comment]
Hi Noel, it looks like a time format of some time dimension is not set in the cube creator. Could you take a look if this is the case and if so, add a time format there and see it this fixes the problem?

I think it should be done via `Data kind -> Time description type -> Time precision`.

**Comment by ortnever** [comment]
Dear Bartosz, I got the same error message with a dataset and I also thought first about this "Time precision" metadata but everything was fine in the Cube Designer.
I filled in other metadata needed for opendata.swiss at the cube's level, such as time coverage and I don't know what else. 
The problem has been solved but I don't know how. Is it not possible to be more precise about this error message?
@NoelLeber : Do you have an other Exemple where we could test and complete the metadata, step by step?

**Reply by bprusinowski** [reply]
Hi Veronique, if it's indeed the case that it was not only the `timeFormat` property that was missing from the cube but adding other fields fixed this particular error, then I agree we should be more precise with the error. It would be great if we had some test cube to find out the steps needed to mitigate the issue and then introduce better error handling. If you or @NoelLeber have such examples to work with, it would be extremely helpful to move forward. Thank you in advance 🙏

**Comment by ortnever** [comment]
Hi @bprusinowski 
Here is an example
Cube on INT : https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!tobiasholzbeschaffungarbeitsstaettencopy-lw81mv7g8km/edit
Dataset in Visualize :  consult please via publication in cube creator.
![image](https://user-images.githubusercontent.com/79834090/195607198-8e6a0fb8-749f-400e-a9c5-8824520d6420.png)


This is very strange as it is an import from Noel's project that we completed (see my comment above) and we have no more this message for the cube of Noel. For this cube there is a problem with the cascading filter but not this error message about the time dimension.
Noel's cube : https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!lnholzbeschaffungarbeitsstaettenneu-dkk1mlanoav/csv-mapping
We made some changes this morning in the Noel's cube (hence the different version) but it was already working before these changes.

**Reply by bprusinowski** [reply]
Hi @ortnever, thanks so much for providing this example! It seems like in this case the dimension type is wrong (Jahr is nominal, while for temporal dimensions it should be interval). The error message about missing timeFormat is indeed misleading here, we'll make it more explicit.

**Reply by bprusinowski** [reply]
The error message has been improved by #784.
